content/en/docs/reference/setup-tools/kubeadm/kubelet-integration.md

```diff
@@ -0,0 +1,88 @@
+## Configuring the kubelets in your cluster with kubeadm and packaging
+
+#### Introduction to the relation between kubeadm and kubelet
+
+The `kubeadm` CLI tool has a lifecycle that is decoupled from the lifecycle of the `kubelet` daemon, the [Kubernetes Node Agent](#TODO). The `kubeadm ` CLI tool is executed by the user at initialization or upgrade time. The kubelet is always running on any given master or node that is running Kubernetes. As the `kubelet` is a daemon, it needs to be run by some kind of init system or process manager. In the deb and rpm packages that are shipped with Kubernetes releases, the `kubelet` is by default run as a `systemd` service. _Important to note_ though, is that _any type_ of init system or process manager can be used for running the `kubelet`, this default solution is a reference solution, but swappable. By default, the `kubeadm` deb/rpm package ships a `systemd` dropin file for the kubelet, so it becomes functional. The vanilla `kubelet` deb/rpm package does not ship with any CLI flags at all (in other words, is non-functional).
```

Comment:
"does not ship with any CLI flags at all" => does not ship with any default values? current phrasing almost sounds like the kubelet has no configurable flags at all